buildings.mss

```diff
@@ -23,7 +23,9 @@
   [zoom >= 13] {
     [aeroway = 'terminal'],
     [amenity = 'place_of_worship'],
-    [building = 'train_station'] {
+    [building = 'train_station'],
+    [aerialway = 'station'],
+    [public_transport = 'station'] {
```

Comment:
Will this hide missing `building=*` on `public_transport=station` ?
I often found this in the past for place_of_worships which missed the `building` tag.

Reply:
In general (excluding building=train_station case of course) we're not able to tell if this is missing building tag or just this is an area. So this will show only buildings with that tag, area rendering for all stations is still to be added in separate PR.